chapters/en/unit2/cnns/yolo.mdx

```diff
@@ -56,21 +114,26 @@ Taking into consideration the timeline and the different licensing of the models
 This short chapter presented the history/evolution of YOLO in a linear way. However, this is not the case—many other versions of YOLO were released in parallel. Notice the release of YOLOv4 and YOLOv5 in the same year. Other versions that we did not cover include YOLOvX (2021) which was based on YOLOv3 (2018), YOLOR (2021) which was based on YOLOv4(2020), and many others.
 Also, It is important to understand that the selection of the ‘best’ model version depends on the user requirements such as speed, accuracy, hardware limitations, and user-friendliness. For example, YOLOv2 is very good at speed. YOLOv3 provides a balance between accuracy and speed. YOLOv4 has the best ability for adapting/being compatible across different hardware.
 
-References:
-
-[1] [YOLO9000: Better, Faster, Stronger](https://arxiv.org/abs/1612.08242) <br />
-[2] [YOLOv3: An Incremental Improvement](https://arxiv.org/abs/1804.02767) <br />
-[3] [YOLOv4: Optimal Speed and Accuracy of Object Detection](https://arxiv.org/abs/2004.10934) <br />
-[4] [YOLOv4 GitHub repo](https://github.com/AlexeyAB/darknet) <br />
-[5] [Ultralytics YOLOv5](https://docs.ultralytics.com/models/yolov5/) <br />
-[6] [YOLOv6: A Single-Stage Object Detection Framework for Industrial Applications](https://arxiv.org/abs/2209.02976) <br />
-[7] [YOLOv6 GitHub repo](https://github.com/meituan/YOLOv6) <br />
-[8] [YOLOv6 v3.0: A Full-Scale Reloading](https://arxiv.org/abs/2301.05586) <br />
-[9] [YOLOv7: Trainable bag-of-freebies sets new state-of-the-art for real-time object detectors](https://arxiv.org/abs/2207.02696) <br />
-[10] [YOLOv7 GitHub repo](https://github.com/WongKinYiu/yolov7) <br />
-[11] [Ultralytics YOLOv8](https://github.com/ultralytics/ultralytics) <br />
-[12] [YOLOv9: Learning What You Want to Learn Using Programmable Gradient Information](https://arxiv.org/abs/2402.13616) <br />
-[13] [YOLOv9 GitHub repo](https://github.com/WongKinYiu/yolov9) <br />
-[14] [YOLOvX](https://yolovx.com/) <br />
-[15] [You Only Learn One Representation: Unified Network for Multiple Tasks](https://arxiv.org/abs/2105.04206)
 
+## Reference 
+
+[1] [Rich feature hierarchies for accurate object detection and semantic segmentation](https://arxiv.org/abs/1311.2524v5)
```

Comment:
I think you should add  <br /> after each reference line in order for every reference to appear in one line.